assets/bundles/bundle.properties

```diff
@@ -191,7 +194,7 @@ setting.cyber-io-always-show-link.description=Whether the arrow always shows up
 setting.cyber-io-show-wireless-tower-circle.name=Show Wireless Tower Circle
 setting.cyber-io-show-wireless-tower-circle.description=Whether the infinite pinging circle will show up
 setting.cyber-io-show-welcome.name=Receive Message from Plum Star Inc.
-setting.cyber-io-show-welcome.description=Whether flavor text shows up when game starts up.
+setting.cyber-io-show-welcome.description=Whether flavourful text shows up when game starts up.
```

Comment:
`flavor text` is borrowed from [Hearthstone](https://hearthstone.fandom.com/wiki/Flavor_text).

Reply:
ok